features/mutation-events.yml

```diff
@@ -4,8 +4,11 @@ description: Mutation events like `DOMSubtreeModified`, `DOMNodeInserted`, or `D
 spec: https://w3c.github.io/uievents/
 group: dom
 discouraged:
+  pending_removal: true
```

Comment:
After some discussion with MDN, it occurred to me that instead of a boolean, we could have something like `removal_scheduled_date: 2025-12-31`. The presence of this field could indicate that removal is pending as well as when we expect it to be removed.

MDN also suggested that the reason field contain a removal timeline in the case of pending removal features.